.travis.yml

```diff
@@ -56,8 +56,18 @@ jobs:
       install: travis_retry composer update --prefer-dist --prefer-lowest
 
     - stage: Test
-      env: DB=sqlite DEPENDENCIES=stable
-      install: travis_retry composer update --prefer-dist --prefer-stable
+      if: type = cron
+      env: DB=sqlite
+      install:
+        - composer config minimum-stability dev
+        - travis_retry composer update --prefer-dist
+
+    - stage: Lint
```

Comment:
Looks like incorrect rebase, this shouldn't be here.

Reply:
Indeed, sorry